src/main/java/org/json/XML.java

```diff
@@ -10,6 +10,9 @@
 import java.math.BigInteger;
 import java.util.Iterator;
 
+import static org.json.NumberConversionUtil.potentialNumber;
```

Comment:
Just import NumberConversionUtil, not the methods.

Reply:
In the next PR it would be changed.